extensions/amp-analytics/amp-analytics.md

```diff
@@ -219,15 +219,16 @@ The `extraUrlParamsReplaceMap` attribute specifies a map of keys and values that
 
 The `triggers` configuration object describes when an analytics request should be sent. The `triggers` attribute contains a key-value pair of trigger-name and  trigger-configuration. A trigger-name can be any string comprised of alphanumeric characters (a-zA-Z0-9). Triggers from a  configuration with lower precedence are overridden by triggers with the same names from a configuration with higher precedence.
 
-  - `on` (required) The event to listener for. Valid values are `render-start`, `ini-load`, `visible`, `click`, `scroll`, `timer`, and `hidden`.
+  - `on` (required) The event to listener for. Valid values are `render-start`, `ini-load`, `click`, `scroll`, `timer`, `visible`, `hidden`, [`amp-access-*`](../amp-access/amp-access-analytics.md), and [`video-*`](./amp-video-analytics.md)
```

Comment:
What specific video component are the analytics for?  If it pertains to `amp-video` then that's what s/b specified.

Reply:
The analytics are for any video extension implementing the `video-interface` interface. I know 8 do, but at this time only `<amp-video>` will give the `currentTime`, `duration` and `playedRangesJson` values in the analytics data. The video components do not yet expose these values. I'm not sure of the cleanest way to communicate that here. Maybe it could be a section in the `amp-video-analytics.md`?